### PR TITLE
cleanup logger

### PR DESCRIPTION
**Pull request by charlierudolph**

### CODE REVIEW

File: src/application/deployer.go [hunk 1]
```diff
@@ -18,17 +18,16 @@ func StartDeploy(deployConfig types.DeployConfig) error {
...
 
-	deployConfig.LogChannel <- "Setting up AWS account..."
+	deployConfig.Logger.Log("Setting up AWS account...")
 	err = aws.InitAccount(deployConfig.AwsConfig)
 	if err != nil {
 		return err
 	}
 
-	deployConfig.LogChannel <- fmt.Sprintf("Building %s %s...\n\n", deployConfig.AppConfig.Name, deployConfig.AppConfig.Version)
+	deployConfig.Logger.Log(fmt.Sprintf("Building %s %s...\n\n", deployConfig.AppConfig.Name, deployConfig.AppConfig.Version))
```
Comment (alexdavid) [inline src/application/deployer.go:27]: Can be a separate PR but would be cool if we provided a `Logf` method like

```go
deployConfig.Logger.Logf("Building %s %s...\n\n", deployConfig.AppConfig.Name, deployConfig.AppConfig.Version)
```

Comment (charlierudolph) [reply src/application/deployer.go:27]: Will do separately

File: src/util/logger.go [hunk 2]
```diff
@@ -32,35 +33,19 @@ func NewLogger(roles, silencedRoles []string, writer io.Writer) *Logger {
...
 	text = strings.TrimSpace(text)
 	for _, line := range strings.Split(text, "\n") {
-		serviceName, serviceOutput := util.ParseDockerComposeLog(role, line)
-		if !util.DoesStringArrayContain(l.SilencedRoles, serviceName) {
+		serviceName, serviceOutput := ParseDockerComposeLog(l.DefaultRole, line)
+		if !DoesStringArrayContain(l.SilencedRoles, serviceName) {
 			err := l.logOutput(serviceName, serviceOutput)
 			if err != nil {
-				return err
+				panic(err)
```
Comment (alexdavid) [inline src/util/logger.go:45]: 👍

**Review by alexdavid — APPROVED** [review]
Awesome! 🚀
